thrift/src/test/java/com/linecorp/armeria/it/server/GracefulShutdownIntegrationTest.java

```diff
@@ -133,39 +145,38 @@ public void testBaseline() throws Exception {
                                                    baselineNanos + MILLISECONDS.toNanos(400));
     }
 
-    @Test(timeout = 20000L)
-    public void waitsForRequestToComplete() throws Exception {
+    @Timeout(20)
+    @Test
+    void waitsForRequestToComplete() throws Exception {
         final long baselineNanos = baselineNanos();
         server.start();
 
         final SleepService.Iface client = newClient();
         final AtomicBoolean completed = new AtomicBoolean(false);
-        final CountDownLatch latch = new CountDownLatch(1);
```

Comment:
This latch does not guarantee the sleep service is called. So I changed to use `AtomicBoolean`.